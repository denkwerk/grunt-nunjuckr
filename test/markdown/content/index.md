# This is a simple markdown file

This file is a part of a test for grunt-nunjuckr.
