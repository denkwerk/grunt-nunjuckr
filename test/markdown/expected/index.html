<!DOCTYPE html>
<html>
<head>
    <meta charset="UTF-8">
    <title>Nunjuckr markdown test</title>
</head>
<body>

    <h1 id="thisisasimplemarkdownfile">This is a simple markdown file</h1>

<p>This file is a part of a test for grunt-nunjuckr.</p>

</body>
</html>
